subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -889,6 +889,13 @@ uint8_t ll_adv_enable(uint8_t enable)
 		conn->supervision_expire = 0;
 		conn->procedure_expire = 0;
 
+#if defined(CONFIG_BT_CTLR_IGNORE_SAME_PEER_CONN)
+		conn->own_addr_type = 0;
+		memset(conn->own_addr, 0xFF, sizeof(conn->own_addr));
+		conn->peer_addr_type = 0;
+		memset(conn->peer_addr, 0xFF, sizeof(conn->peer_addr));
```

Comment:
What's the reasoning behind setting the addresses to 0xFFFFFF?

Reply:
0xFFFFFF matches `BT_ADDR_NONE`, which I find slightly more intuitive than 0x000000, which would match `BT_ADDR_ANY`.

Reply:
I see. In the case that a literal value (in this case 0xFFFFFF) has a specific meaning (e.g. to be comparable with BT_ADDR_NONE), then I strongly suggest that we add a `#define` or similar for the value.

Reply:
Copying from `BT_ADDR_LE_NONE` now to make the meaning clearer.